asc2018/vdr/electrical/WaterlooMidnightSun_ASC2018VDR_BatteryTechReport.tex

```diff
@@ -0,0 +1,119 @@
+\newcommand\theuniversityname{University of Waterloo} % should also not change normally
+\newcommand\theteamwebsite{www.uwmidsun.com} % should also not normally change
+\newcommand\theteamphone{(519) 888-4567 x32978} % should also not normally change
+
+\newcommand\thetitle{Battery Tech Report} % <--------------- add the title
+\newcommand\thesubtitle{Electrical} % <--------------- add a subtitle or leave the field empty
+\newcommand\theauthor{Minghao Ji} % <--------------- add an author with contact info or comment this line
+\newcommand\theauthorcontact{minghao.ji@uwmidsun.com} % <--------------- add author's email or leave the field empty
+\newcommand\thedate{\today} % <--------------- update the date, use this format
+
+\pagestyle{fancy}
+\renewcommand{\headrulewidth}{0pt}
+\lhead{\thetitle}
+\rhead{\theteamname}
+
+\begin{document}
+
+% Title Page
+\begin{titlepage}
+\large
+\vspace*{2cm}
+\centering
+\includegraphics[width=.25\textwidth]{./figures/midnightSunLogoCircle.png} \\
+\vspace{1.5cm}
+{\LARGE \theteamname} \\
+\theuniversityname \\
+\vspace{2.2cm}
+{\LARGE MSXII} \\
+\vspace{0.4cm}
+{\huge\bfseries \thetitle} \\
+\vspace{0.2cm}
+{\LARGE \thesubtitle} \\
+\vspace{2.2cm}
+\ifdefined \theauthor
+\par Prepared by: \\
+\theauthor \\
+\theauthorcontact \\
+\fi
+\thedate \\
+\vfill
+\theteamwebsite \\
+\theteamphone
+\end{titlepage}
+
+% Main Matter
+\tableofcontents
+%\listoffigures % <-------------- uncomment for list of figures
+%\listoftables % <-------------- uncomment for list of tables
+
+\section{Battery Specifications}
+
+\begin{table}[!htbp]
+\centering
+\begin{tabular}{|l|l|l|}
+\hline
+\multirow{4}{*}{5.2.D.1} & Manufacturer Name                     & LG Chemical Limited                                                                                                                   \\ \cline{2-3}
+                         & Manufacturer Contact                  & \begin{tabular}[c]{@{}l@{}}Jon Caserta, Liion Wholesale Batteries (Supplier)\\ support@liionwholesale.com\\ 888-972-2883\end{tabular} \\ \cline{2-3}
+                           & Cell Specification URL                & \url{http://vruzend.com/SpecificationINR18650MJ1.pdf}                                                                                       \\ \cline{2-3}
+                         & Cell MSDS URL                         & \url{http://bigship.com/telechargement?doc=148635}                                                                                          \\ \hline
+5.2.D.2                  & Stock Number                          & INR18650 MJ1                                                                                                                          \\ \hline
+5.2.D.3                  & Cell \& Module Voltage                & \begin{tabular}[c]{@{}l@{}}3.635 V (nominal)\\ 4.200 V (max)\end{tabular}                                                               \\ \hline
```

Comment:
Do we have a min? If so we may as well include it. Most of the time in datasheets this is laid out as: